schematic/store/synapse.py

```diff
@@ -1903,7 +1913,17 @@ async def add_annotations_to_entities_files(
             # Merge dataframes to add entityIds
             manifest = manifest.merge(
                 file_df, how="left", on="Filename", suffixes=["_x", None]
-            ).drop("entityId_x", axis=1)
+            )
+
+            # drop the duplicate entity column with NA values
+            col_to_drop = "entityId_x"
```

Comment:
To add context @linglp , after the noted issue was addressed I ran into another issue with the user's manifest where the column containing the entityIds was dropped after the merge, this ensures that other groups wont run into that issue. @thomasyu888 should I open a ticket to track that work as well?

Reply:
From my perspective, the problem of "the user's manifest where the column containing the entityIds were dropped after the merge" is a separate problem than get_node_validation_rules wasn't called with appropriate parameters. If resolving FDS-2268 means resolving both, we could use the same ticket. But if the merging problem is separate from FDS-2268, there should be a separate ticket.

Reply:
This turns out to be a different issue. Here's the new ticket: https://sagebionetworks.jira.com/browse/FDS-2378

Reply:
Thanks @GiaJordan and @linglp!  Yes, agreed that it should be tracked in a separate ticket and we can bring it up during sprint planning for prioritization.